airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/default_file_based_stream.py

```diff
@@ -148,6 +149,8 @@ def get_json_schema(self) -> JsonSchema:
     def _get_raw_json_schema(self) -> JsonSchema:
         if self.config.input_schema:
             schema = self.config.input_schema
+        elif self.config.schemaless:
```

Comment:
is it possible for both input_schema and schemaless to be set at the same time? It's not obvious to me that the input_schema should take precedence. Can we fail or at least log a warning that this is unexpected?